features/hsl.yml

```
@@ -0,0 +1,13 @@
+name: HSL
+description: The HSL color model specifies colors using hue, saturation and lightness channels. It can be specified using `hsl()` and `hsla()` functions.
```

We should make it clear we're talking about CSS here, specifically.

Also, "specifies" is sometimes unclear and there are usually better alternatives (see [the guideline entries relating to this](https://github.com/web-platform-dx/web-features/blob/main/docs/guidelines.md#specifies)). I think "pick" is a nice choice (aligning with color pickers, for instance) for colors:

```suggestion
description: The `hsl()` and `hsla()` CSS functions pick colors using hue, saturation and lightness channels.
```